### ISSUE TITLE
Cannot Unlock Ice Splash

### ISSUE THREAD

**Issue by AzraelNewtype** [larsiusprime/tdrpg-bugs#1335]
# Basic Info
```
Linux
Download
Full Version
2.1.7
Windowed 1152x864
```

# Steps to Reproduce
1. Reach Level 15 with Wrenna (haven't recruited a generic)
1. Bring mouse over to fourth skill bubble in party menu

# What I Expected to Happen
The mouse would continue into the bubble, triggering a tooltip and allowing me to click.

# What Happened Instead
The game ceased responding to input, including SIGTERM. Needed to resort to SIGHUP to close the window successfully. Activating crash logging before reproducing doesn't appear to have generated anything, though I might just not know where to find any extant log.

![defendersquesti_001](https://user-images.githubusercontent.com/1253200/29759698-e54f8fcc-8b8b-11e7-98df-255628953fe6.png)

The OS cursor did not reappear over the window, but input stops roughly there every time.

**Comment by toni3m** [comment]
```
Operating System: Linux
Browser/Download: Download
Demo/Full?      : Full Version
Version #       : 2.1.7 (gog.com)
Optional Info   : various resolutions in windowed mode
```
I reproduce the bug if selected resolution is **1024x768, 1152x864 or 1280x960**.
When using one of the following resolutions, the information for Ice Splash appears as expected (albeit with wrong text placement, and target value is "$MISC_SPLASH_TILES", probably related to bug  #784): 800x600, 960x544, 1024x600, 1152x648, 1280x720, 1366x768, 1600x900 (I did not try every available resolution).

Please note the same bug affects Dragons' **Fire Ball** (tested with the same set of resolutions).

**Comment by larsiusprime** [comment]
Thanks, looking into this now.

**Comment by larsiusprime** [comment]
I believe I have fixed this bug in the 2.2.1 hotfix, which is due out soon.

**Comment by larsiusprime** [comment]
@AzraelNewtype Try the new hotfix, up on test_public now. Reopen if there's any issues.
